src/pages/Home/sections/Prototype/components/Link/index.jsx

```diff
@@ -1,4 +1,5 @@
 import { Link } from 'react-router-dom'
+import PrototypeIcon from '@components/UiKit/Icons/Prototype/index.jsx'
```

Comment:
The NewsLetter icon or Prototype icon can be reused for both Hero and Prototype sections; it may be beneficial to replace PrototypeIcon with NewsLetterIcon and remove the duplicate.